libs/nestjs-admin/src/admin.controller.ts

```diff
@@ -31,6 +44,8 @@ type AdminModelsResult = {
 }
 
 @Controller('admin')
+@UseGuards(AdminGuard)
+@UseFilters(new AdminFilter())
```

Comment:
Not 100% sure but I think you can 
```suggestion
@UseFilters(AdminFilter)
```